apps/heft/src/cli/HeftCommandLineParser.ts

```diff
@@ -2,7 +2,12 @@
 // See LICENSE in the project root for license information.
```

Comment:
In your PR, could you add `heft start` definitions to `heft-web-rig` and `heft-node-rig`? This way we can see a fully worked out example.